cache/cache.go

```diff
@@ -22,12 +22,13 @@ import (
 	"sync"
 	"time"
 
+	pb "go.etcd.io/etcd/api/v3/etcdserverpb"
 	rpctypes "go.etcd.io/etcd/api/v3/v3rpc/rpctypes"
 	clientv3 "go.etcd.io/etcd/client/v3"
 )
 
+// TODO: add gap-free replay for arbitrary startRevs and drop this guard.
```

Comment:
@apullo777 see if you can revert this comment change. Ideally it shouldn't show up as a diff here.